swarm_test.go

```diff
@@ -109,19 +108,6 @@ func TestNewSwarm(t *testing.T) {
 				}
 			},
 		},
-		{
```

Comment:
Me too.

In fact on first glance (I only did a quick check) it seems as it is not possible to pass the initial amount via config variable anymore, and this tests would probably catch it!

Reply:
We had a conversation about this in the incentives channel.

Putting this code back will result in an error on line 155 in `swap.go`
``` golang	
networkID, err := backend.NetworkID(context.TODO())
```

As you can see, the test creates an ipc endpoint with no registered api on it (`swarm.go:58`). Hence, a call to backend.NetworkID will result in an error. Backend.networkID is low lever an rpc call with `net_version` as parameter. It will return a NetworkID as a string. 

We can put this test only back when we can mock this behavior. I don't know how to do it, but I'm sure it is possible. I propose to make an issue and do this in a follow-up PR.

Reply:
Yes please, let's not skip tests lightly because something stops working. As we want to merge this PR quickly I agree to create an issue and tackle in follow-up PR.

Reply:
While creating the issue, I realize that even when making the ipc-backend to respond to a `net_version` call, the new design won't allow us to do a test on creating a new Swarm with swap enabled, because the call to `swap.New()` now needs a working backend. 

I am still making the issue for future reference, but I suspect that solving this one won't be very easy

Reply:
https://github.com/ethersphere/swarm/issues/1820

Reply:
Yes this will be a bit harder to address but we have to do it. It may need some redesign.

Important is that we now have a good and stable release for devcon.